spec/features/user_spec.rb

```diff
@@ -16,9 +16,17 @@
 
       expect(page).to have_content("New user created.")
     end
-  end
 
-  describe Pages::Users do
+    it "#edit_user" do
+      session = Pages::Users.new
+      session.open_main_page
+      session.visit_users_page
```

Comment:
I have noticed, that this part repeats for another scenario, so it can be moved to before(:each) hook. You can read more under the link https://www.tutorialspoint.com/rspec/rspec_hooks.htm

Reply:
Thanks! Here is a good example of using the "DRY" principle in my code.